packages/bitstring/bitstring.2.1.0/opam

```diff
@@ -31,7 +31,7 @@ You can use this module to both parse and generate binary formats, files and pro
 Bitstring handling is added as primitives to the language, making it exceptionally simple to use and very powerful."""
 flags: light-uninstall
 url {
-  src: "https://bitbucket.org/thanatonauts/bitstring/get/v2.1.0.tar.gz"
+  src: "https://github.com/xguerin/bitstring/archive/v2.1.0.tar.gz"
```

Comment:
Thanks to @kit-ty-kate, I persisted and found a source via the md5 has on ocamlpro's cache:

```suggestion
  src: "https://gitlab.ocamlpro.com/OCamlPro/opam-repository/-/raw/cached/cache/md5/70/70f7acebe337072bff03a1f7a3fb1d3a"
```

https://gitlab.ocamlpro.com/OCamlPro/opam-repository/-/raw/cached/cache/md5/70/70f7acebe337072bff03a1f7a3fb1d3a